artemis-server/src/main/java/org/apache/activemq/artemis/core/server/ActiveMQServerLogger.java

```diff
@@ -1711,7 +1711,7 @@ void slowConsumerDetected(String sessionID,
    @Message(id = 224030, value = "Could not cancel reference {0}", format = Message.Format.MESSAGE_FORMAT)
    void errorCancellingRefOnBridge(@Cause Exception e, MessageReference ref2);
 
-   @LogMessage(level = Logger.Level.ERROR)
```

Comment:
I agree it should be DEBUG: Can you remove this logger, and just call log.debug(.. ) where this is being called?

Reply:
Done (untested)